pynestml/meta_model/ast_expression.py

```diff
@@ -313,59 +316,48 @@ def get_function_calls(self):
             ret.extend(self.get_if_not().get_function_calls())
         return ret
 
-    def get_parent(self, ast=None):
-        """
-        Indicates whether a this node contains the handed over node.
-        :param ast: an arbitrary meta_model node.
-        :type ast: AST_
-        :return: AST if this or one of the child nodes contains the handed over element.
-        :rtype: AST_ or None
+    def get_children(self) -> List[ASTNode]:
+        r"""
+        Returns the children of this node, if any.
+        :return: List of children of this node.
         """
         if self.is_expression():
-            if self.get_expression() is ast:
-                return self
-            if self.get_expression().get_parent(ast) is not None:
-                return self.get_expression().get_parent(ast)
+            return [self.get_expression()]
+
         if self.is_unary_operator():
-            if self.get_unary_operator() is ast:
-                return self
-            if self.get_unary_operator().get_parent(ast) is not None:
-                return self.get_unary_operator().get_parent(ast)
+            return [self.get_unary_operator()]
```

Comment:
In the case of a unary operator, should the expression associated with the operator be included in the children?